master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Felix Configuration Resource (FelixConfiguration)
+---
+
+A Felix configuration resource (FelixConfiguration) represents Felix configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `felixconfiguration`, `felixconfig`, `felixconfigurations`, `felixconfigs`.
+
+See [Configuring Felix]({{site.baseurl}}/{{page.version}}/referance/felix/configuration) for more details.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  ipv6Support: false
+  ignoreLooseRPF: true
+  chainInsertMode: Append
+```
+
+### BGP Peer Definition
```

Comment:
I am not sure.......does Felix have a BGP Peer definition? I don't see anything about this in the Felix configuration docs.

Reply:
oops